Packages/vcs/Lib/VTKPlots.py

```diff
@@ -405,6 +408,12 @@ def portrait(self, W, H, x, y, clear):
         else:
             self.renWin.SetSize(W, H)
 
+    def portrait(self, W=-99, H=-99, x=-99, y=-99, clear=0):
+        self.resize_or_rotate_window(W, H, x, y, clear)
+
```

Comment:
@sankhesh you might want to add this as a test too if @aashish24 thinks it is a worthy test :wink: